tests/conftest.py

```diff
@@ -177,3 +182,12 @@ def interval_query(bed_interval) -> Dict:
         "start": bed_interval[1],
         "end": bed_interval[2],
     }
+
+
+@pytest.fixture(name="real_d4_query")
+def real_d4_query(real_coverage_path) -> Dict:
```

Comment:
Add type hint
```suggestion
def real_d4_query(real_coverage_path) -> Dict[str, Path]:
```
or is the second a str?

Reply:
I think it's a string, but I'll check